src/tiled/mapdocumentactionhandler.cpp

```diff
@@ -425,6 +428,14 @@ void MapDocumentActionHandler::cropToSelection()
     mMapDocument->resizeMap(bounds.size(), -bounds.topLeft(), true);
 }
 
+void MapDocumentActionHandler::autocrop()
+{
+    if (!mMapDocument)
+        return;
+
+    mMapDocument->autocropMap();
```

Comment:
Please write it like this, mostly because it's shorter and consistent with other functions in this file:
```c++
if (mMapDocument)
    mMapDocument->autocropMap();
```
The reason why I often will do an "early exit" like you did here is to avoid indenting a lot of code or to make the function more readable, but that does not apply here.